.github/workflows/bump_project_version.yml

```diff
@@ -0,0 +1,42 @@
+#############################################################################
+# GitHub Action to Bump Project Version.
```

Comment:
bump_project_version.yml -> bump-cs-version.yml 

>GitHub Action to Bump Project Version.

GitHub Action to Bump Checkstyle Version in repository.

Reply:
Changes Done